### PR TITLE
Added a handy translate button (old #220)

### PR DESCRIPTION
**Pull request by tht7**
# Checklist
- [x] I have described what this PR contains

**Choose one of the following two options:**
- - [x] This PR does not introduce major changes
- - [ ] This PR introduces major changes, and I have consulted @buresdv, @jboi or @mormaer in the *Mlem Development Matrix room*

**Choose one of the following two options:**
- - [ ] This PR does not change the UI in any way
- - [x] This PR adds new UI elements / changes the UI, and I have attached pictures or videos of the new / changed UI elements

# Pull Request Information

## About this Pull Request
- added a cute "Translate" button that takes you to google translate with the text already pre-inserted (and also hiding google translate of your phones language)

## Screenshots and Videos
IMG_7200

https://github.com/mormaer/Mlem/assets/6067331/70e52269-4a3a-4b46-86b1-ed9292821ea6

## Additional Context
**Any additional context you'd like to add to help us review this PR**
IMG_7201
IMG_7199

### CODE REVIEW

**Review by WestonHanners — APPROVED** [review]

File: Mlem/Extensions/Translation Shower.swift [hunk 1]
```diff
@@ -0,0 +1,20 @@
+//
+//  Translation Shower.swift
```
Comment (EricBAndrews) [inline Mlem/Extensions/Translation Shower.swift:2]: Shower me in translations

File: Mlem/Views/Tabs/Posts/Components/Comment/Components/Comment Item Logic.swift [hunk 2]
```diff
@@ -223,7 +223,16 @@ extension CommentItem {
             enabled: true) {
                 isComposingReport = true
             })
-                   
+
+        // translate
+        ret.append(MenuFunction(
+            text: "Translate",
+            imageName: "globe",
```
Comment (EricBAndrews) [inline Mlem/Views/Tabs/Posts/Components/Comment/Components/Comment Item Logic.swift:230]: Globe is already used for websites--maybe `globe.asia.australia`?

File: Mlem/Views/Tabs/Posts/Components/Comment/Components/Comment Item.swift [hunk 3]
```diff
@@ -179,6 +180,9 @@ struct CommentItem: View {
             } else if !isCollapsed {
                 MarkdownView(text: hierarchicalComment.commentView.comment.content, isNsfw: hierarchicalComment.commentView.post.nsfw)
                     .frame(maxWidth: .infinity, alignment: .topLeading)
+                    .contextMenu {
+                        EasyTranslateButton(text: .constant(hierarchicalComment.commentView.comment.content))
+                    }
```
Comment (EricBAndrews) [inline Mlem/Views/Tabs/Posts/Components/Comment/Components/Comment Item.swift:185]: CommentItem already has a context menu on it--can this be integrated?

Might be a byproduct of this PR being ancient lol

File: Mlem/Views/Tabs/Posts/Components/Post/Components/Community View/Components/Sidebar.swift
Comment (EricBAndrews) [inline Mlem/Views/Tabs/Posts/Components/Post/Components/Community View/Components/Sidebar.swift:1]: Empty file?

**Review by EricBAndrews — CHANGES_REQUESTED** [review]

**Review by EricBAndrews — APPROVED** [review]

**Pull request merged**